configure.bat

```diff
@@ -108,18 +110,21 @@ exit /B 0
     @echo Here %NL%we go
     @echo Meson flags: %MESONFLAGS:        =!NL!%
     @echo ------------------------------
+    @echo Extra flags: %MESONFLAGS_EXTRA%
+    @echo ------------------------------
+    @set MESONFLAGS=%MESONFLAGS:            = %
 exit /B 0
 
 :generate_build
     @echo ------------------------------
     @echo Generating build...
-    set vcvars64="C:\Program Files (x86)\Microsoft Visual Studio\2019\Professional\VC\Auxiliary\Build\vcvars64.bat"
+    set vcvars64="C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Auxiliary\Build\vcvars64.bat"
```

Comment:
This change is needed?

Reply:
PFF WOW Accidental actually :D Maybe we should check for VS version when doing this, hm...

Reply:
Yeah, seems like a good idea